### ISSUE TITLE
v0.4.0 cant view image details

### ISSUE THREAD

**Issue by mms-gianni** [klustair/klustair-frontend#81]
In some cases it ist not possible to open the image details view. 

https://klustair.yourdomain.com/image/9abf57c5-6a75-412b-a700-d97bf30bc8b2/fb821d16-3c7f-4068-96fd-b3e96f0c490b

```
Undefined index: targets (View: /var/www/resources/views/image.blade.php)
```

**Comment by mms-gianni** [comment]
Not sure if same or other Bug. 

```
10.244.XX.XX - - [02/Aug/2021:11:02:49 +0000] "POST /api/v1/pac/report/941fc009-ab18-4ad3-84a1-3a04acfc67f4/image/create HTTP/1.1" 200 7841 "-" "python-requests/2.23.0"
[2021-08-02 11:02:51] locallocal.ERROR: Allowed memory size of 134217728 bytes exhausted (tried to allocate 10035200 bytes) {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\FatalError(code: 0): Allowed memory size of 134217728 bytes exhausted (tried to allocate 10035200 bytes) at /var/www/vendor/facade/ignition/src/ErrorPage/ErrorPageViewModel.php:144)
[stacktrace]
#0 {main}
"}
10.244.XX.XX  - - [02/Aug/2021:11:02:50 +0000] "POST /api/v1/pac/report/941fc009-ab18-4ad3-84a1-3a04acfc67f4/vuln/create HTTP/1.1" 500 2808 "-" "python-requests/2.23.0"
```

**Comment by mms-gianni** [comment]
Something is going wrong while saving the Report. 

ErrorException
Trying to get property 'fulltag' of non-object

```
        $vulnhistory_sql = <<<SQL

            to_char(k_reports.checktime, 'DD.MM HH24:MI') as checktime,

            (SELECT

                COUNT(k_vuln_trivy.uid) AS total

                FROM k_vuln_trivy

                LEFT JOIN k_images ON image_uid = k_images.uid

                WHERE fulltag='$image->fulltag' and

                k_vuln_trivy.report_uid=k_reports.uid

            )

        SQL;
```

**Comment by mms-gianni** [comment]
This issue should be solved now in v0.4.0
